crawler/rabbit/background_consumer.py

```diff
@@ -47,7 +51,9 @@ def _maybe_reconnect(self):
             self._consumer_var = None
 
     def _get_reconnect_delay(self):
-        if self._consumer.was_consuming:
+        if self._consumer.had_transient_error:
+            self._reconnect_delay = 30
+        elif self._consumer.was_consuming:
             self._reconnect_delay = 0
         else:
             self._reconnect_delay += 1
```

Comment:
Why is this increment of 1?

Reply:
I didn't write most of this, as it was provided by Pika, but from what I can understand, the only time you might be incrementing 1 seconds at a time is upon first connection while it is trying to get connected to Rabbit.  Then after it has connected `was_consuming` is set to `True` so you always just try the connection again immediately and repeatedly if it drops.  I've added the extra condition at the top for when we had a transient error which wasn't because of RabbitMQ dropping the connection, but something downstream.  Then I'm making it always wait 30 seconds at a time until the resources become available again and the head message gets correctly processed.